packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -22,7 +23,16 @@ ${success('Success!')} Created a new Contentful app in ${highlight(tildify(folde
 
   wrapInBlanks(highlight('---- Next Steps'))
 
-  console.log(`Now kick it off by running
+  console.log(`Now create an app definition for your app by running
+
+    ${code(`cd ${tildify(folder)}`)}
+    ${code(useYarn ? 'yarn create-app-definition' : 'npm run create-app-definition')}
+
+    or you can create it manually in web app:
```

Comment:
It's now two "or"s chained. Maybe we can improve the wording a bit to avoid that as it doesn't read smooth

Reply:
Good point. Fixed!